src/Umbraco.Infrastructure/Packaging/PackageDataInstallation.cs

```diff
@@ -121,9 +123,10 @@ public InstallationSummary InstallPackageData(CompiledPackage compiledPackage, i
         /// </summary>
         /// <param name="docTypeElements">Xml to import</param>
         /// <param name="userId">Optional id of the User performing the operation. Default is zero (admin).</param>
+        /// <param name="entityContainersInstalled">List of entity containers installed by the package to be populated with those created in installing data types.</param>
         /// <returns>An enumerable list of generated ContentTypes</returns>
-        public IReadOnlyList<IMediaType> ImportMediaTypes(IEnumerable<XElement> docTypeElements, int userId)
-            => ImportDocumentTypes(docTypeElements.ToList(), true, userId, _mediaTypeService);
+        public IReadOnlyList<IMediaType> ImportMediaTypes(IEnumerable<XElement> docTypeElements, int userId, List<EntityContainer> entityContainersInstalled)
```

Comment:
This is a breaking change, we need to old signature reintroduced and obsoleted

Reply:
Yes, I prefer the `out` idea I think - so will change to that.